### PR TITLE
Yaroslav Vysotskyi IM-32 Exam

### PR DESCRIPTION
**Pull request by Yaroslav702**

### CODE REVIEW

File: Tasks/longest.test.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        self.assertEqual(result, 'Roma')
+
+    def test_longest(self):
+        input_array = ['Roma']
+        result = get_longest_arr_string(input_array)
+        self.assertEqual(result, 'Roma')
+
+
+if __name__ == '__main__':
+    unittest.main()
```
Comment (tshemsedinov) [inline Tasks/longest.test.py:33]: All files in git should have one extra empty line at EOF

File: Tasks/month.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  'jul',
+  'aug',
+  'sep',
+  'oct',
+  'nov',
+  'dec'
+]
+
+def get_month_number(month_name: str) -> int:
+    month_prefix = month_name.lower()[:3]
```
Comment (tshemsedinov) [inline Tasks/month.py:19]: 3 is a magic number, use constants

File: Tasks/size.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    converted_size = '0 byte'
+
+    if size != 0:
+        exponent = size.bit_length() // 10
+        converted_size = calculate_size_from_exp(size, exponent)
+
+    return converted_size
+
+
+def calculate_size_from_exp(size: int, exponent: int) -> str:
```
Comment (tshemsedinov) [inline Tasks/size.py:11]: It is better to use dict instead of match/case

File: Tasks/month.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+    return MONTHS.index(month_prefix) + 1 if month_prefix in MONTHS else -1
```
Comment (tshemsedinov) [inline Tasks/month.py:21]: Long line, it's ok but I'd prefer to decompose

File: Tasks/range.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+def _range(Range: tuple) -> list:
```
Comment (tshemsedinov) [inline Tasks/range.py:1]: All identifiers in this example can be shorter without loss of meaning

File: Tasks/longest.test.py (hunk 1 above)
Comment (Yaroslav702) [reply Tasks/longest.test.py]: Fixed.

File: Tasks/month.py (hunk 2 above)
Comment (Yaroslav702) [reply Tasks/month.py]: Fixed.

File: Tasks/size.py (hunk 3 above)
Comment (Yaroslav702) [reply Tasks/size.py:15]: Fixed.

File: Tasks/range.py (hunk 5 above)
Comment (Yaroslav702) [reply Tasks/range.py]: Fixed.

**Pull request closed without merging**